.travis.yml

```diff
@@ -35,6 +35,11 @@ notifications:
       - "chat.freenode.net#oppia"
     on_success: never
     on_failure: always
+  webhooks:
+    urls:
+      - https://webhooks.gitter.im/e/f8f782497ec1ffcea2e7
```

Comment:
Where did you find it? (It might be worth adding a comment linking to the doc that tells you how to find it...)